README.md

```diff
@@ -20,20 +20,22 @@ Jump in: [![Slack Status](http://slack.projecthydra.org/badge.svg)](http://slack
   * [What is Hyrax?](#what-is-hyrax)
     * [Feature list](#feature-list)
   * [Help](#help)
-  * [Getting started](#getting-started)
+  * [Getting Started](#getting-started)
     * [Prerequisites](#prerequisites)
-      * [Characterization](#characterization)
-      * [Derivatives](#derivatives)
+      * [FITS](#fits)
+      * [LibreOffice](#libreoffice)
     * [Environments](#environments)
     * [Ruby](#ruby)
     * [Redis](#redis)
     * [Rails](#rails)
       * [JavaScript runtime](#javascript-runtime)
-  * [Creating a Hyrax\-based app](#creating-a-hyrax-based-app)
-    * [Generate a work type](#generate-a-work-type)
+  * [Quick Start for a new Hyrax app](#quick-start-for-a-new-hyrax-app) 
+    * [Create a Hyrax\-based app](#create-a-hyrax-based-app)
```

Comment:
Can you rename this section header to its former name? (IOW, please revert lines 32 and 33.)